_pytest/python.py

```diff
@@ -933,8 +933,9 @@ def _idval(val, argname, idx, idfn, config=None):
             pass
 
     if config:
-        hook_id = config.hook.pytest_make_parametrize_id(config=config, val=val)
-        if hook_id:
+        hook_id = config.hook.pytest_make_parametrize_id(
+            config=config, val=val, argname=argname)
+        if hook_id is not None:
```

Comment:
this is a pretty tricky behavior change (even tough its unintended behavior)
it needs a test and possibly a discussion (its a change that requires user code to change in case it does accidentally return false looking values, as that is now limited to `None` instead of any false value